### PR TITLE
[zh] Sync Compatibility Versions into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync Compatibility Versions into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+keywords: [profiles,install,helm]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+Istio 的每个新版本都可能会出现一些有意而为的行为变更。
+这些变更可能是为了提高安全性、修复不正确的行为或以其他方式为用户改进 Istio。
+一般来说，这些类型的变更仅影响边缘情况。
+
+虽然从长远来看是有益的，但每次行为改变都会在升级过程中带来风险。
```
Comment (hanxiaop) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:14]: Unify translation of `behavioral change`
```suggestion
虽然从长远来看是有益的，但每次行为变更都会在升级过程中带来风险。
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+title: 兼容版本
+description: 如何配置“兼容版本”，来将行为变更与版本解耦。
+weight: 36
+keywords: [profiles,install,helm]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+Istio 的每个新版本都可能会出现一些有意而为的行为变更。
+这些变更可能是为了提高安全性、修复不正确的行为或以其他方式为用户改进 Istio。
```
Comment (hanxiaop) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:11]:
```suggestion
这些变更可能是为了提高安全性、修复不正确的行为或以其他方式改进 Istio 用户的体验。
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+{{< tab name="Helm" category-value="helm" >}}
+
+{{< text shell >}}
+$ helm install ... --set compatibilityVersion={{< istio_previous_version >}}
+{{< /text >}}
+
+{{< /tab >}}
+{{< /tabset >}}
+
+## 我什么时候应该使用兼容版本？ {#when-should-i-use-compatibility-versions}
```
Comment (hanxiaop) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:43]:
```suggestion
## 什么时候应该使用兼容性版本？ {#when-should-i-use-compatibility-versions}
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+虽然从长远来看是有益的，但每次行为改变都会在升级过程中带来风险。
+从以往经验上看，升级时用户应该通过阅读发布说明来了解任何行为变更并确定它们是否会产生影响；
+这可能很乏味并且容易出错。
+
+兼容性版本为用户提供了额外的选项，允许发布版本与行为变更解耦。
+例如，您可以安装 Istio {{< istio_version >}}，
+但将其配置为类似于 {{< istio_previous_version >}}。
+
+## 使用兼容版本 {#using-compatibility-versions}
```
Comment (hanxiaop) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:22]: `Compatibility version` 的翻译也可以统一一下

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+---
+title: 兼容版本
+description: 如何配置“兼容版本”，来将行为变更与版本解耦。
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:3]:
```suggestion
description: 配置“兼容版本”将行为变更与版本发布解耦。
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+weight: 36
+keywords: [profiles,install,helm]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+Istio 的每个新版本都可能会出现一些有意而为的行为变更。
+这些变更可能是为了提高安全性、修复不正确的行为或以其他方式为用户改进 Istio。
+一般来说，这些类型的变更仅影响边缘情况。
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:12]:
```suggestion
一般来说，这些类型的变更仅影响少量场景。
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md (hunk 1 above)
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:14]:
```suggestion
虽然从长远来看是有益的，但每次行为变更都会引入升级风险。
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+虽然从长远来看是有益的，但每次行为改变都会在升级过程中带来风险。
+从以往经验上看，升级时用户应该通过阅读发布说明来了解任何行为变更并确定它们是否会产生影响；
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:15]:
```suggestion
从以往经验上看，升级时用户应该通过阅读发布说明来了解所有的行为变更并确定自己是否会受到影响；
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+这可能很乏味并且容易出错。
+
+兼容性版本为用户提供了额外的选项，允许发布版本与行为变更解耦。
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:18]:
```suggestion
兼容版本为用户提供了额外的选项，允许发布版本与行为变更解耦。
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+例如，您可以安装 Istio {{< istio_version >}}，
+但将其配置为类似于 {{< istio_previous_version >}}。
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:20]:
```suggestion
但将其行为配置为类似于 {{< istio_previous_version >}}。
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+$ helm install ... --set compatibilityVersion={{< istio_previous_version >}}
+{{< /text >}}
+
+{{< /tab >}}
+{{< /tabset >}}
+
+## 我什么时候应该使用兼容版本？ {#when-should-i-use-compatibility-versions}
+
+建议仅在发现不兼容时才使用兼容版本，而不是默认使用。
+每个兼容性版本只会在几个发布版本中持续，因此最终您将需要迁移到新的行为。
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:46]:
```suggestion
每个兼容版本只会持续两三个发布版本，因此最终您将需要迁移到新的行为。
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+目前，每个兼容性版本都打算保留至少两个版本，尽管这也可能会发生变化。
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:47]:
```suggestion
目前，每个兼容版本打算保留至少两个版本，尽管这也可能会发生变化。
```

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+为了帮助检测是否应使用兼容性版本，可以将
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/compatibility-versions/index.md:49]:
```suggestion
为了帮助检测是否应使用兼容版本，可以将
```

**Review by windsonsea — COMMENTED** [review]
It's recommended to pick one from 兼容版本 or 兼容性版本 for compatibility versions from top to down.

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md (hunk 4 above)
Comment (windsonsea) [reply content/zh/docs/setup/additional-setup/compatibility-versions/index.md:22]: +1

Comment (wilsonwu) [reply content/zh/docs/setup/additional-setup/compatibility-versions/index.md:22]: 目前是全部翻译成 兼容版本，开始再想要不要翻译成版本兼容性，大家可以看看哪个好。

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/docs/setup/additional-setup/compatibility-versions/index.md]: Applyed another suggestion for this line.

File: content/zh/docs/setup/additional-setup/compatibility-versions/index.md (hunk 4 above)
Comment (hanxiaop) [reply content/zh/docs/setup/additional-setup/compatibility-versions/index.md:22]: Prefer 兼容版本

Comment (windsonsea) [reply content/zh/docs/setup/additional-setup/compatibility-versions/index.md:22]: Seems consistent and good now.

**Review by windsonsea — APPROVED** [review]
Thanks, lgtm

**Pull request merged**